extracts/v_extracts.mdcps_aces_survey.sql

```diff
@@ -1,35 +1,37 @@
 USE gabby
 GO
 
-CREATE OR ALTER VIEW extracts.mdcps_aces_survey AS
+--CREATE OR ALTER VIEW extracts.mdcps_aces_survey AS
+
+SELECT s.job_title
+      ,s.[location]
+      ,s.position_status
+      ,s.termination_date
+      ,s.first_name
+      ,s.last_name
+      ,s.position_id
+      ,s.business_unit
+      ,s.employee_number
+      ,s.original_hire_date
+      ,s.flsa AS payclass
+      ,s.address_street
+      ,s.address_city
+      ,s.address_state
+      ,s.address_zip
+      ,s.annual_salary
+      ,s.education_level
 
-SELECT sr.job_title 
-      ,sr.[location]
-      ,sr.position_status
-      ,sr.first_name
-      ,sr.last_name
-      ,sr.position_id
-      ,sr.business_unit
-      ,sr.employee_number
-      ,sr.original_hire_date
       ,cf.[Miami - ACES Number] AS miami_aces
-      ,sr.flsa AS payclass
+
       ,'2x Month' AS pay_frequency
-      ,NULL AS highest_education_level
-      ,NULL AS duty_days
+      ,'' AS duty_days
       ,'N/A' AS teacher_eval
-      ,sr.address_street
-      ,sr.address_city
-      ,sr.address_state
-      ,sr.address_zip
-      ,NULL AS MonthlyCarrierCost
-      ,NULL AS MedicalCoverage
       ,'N/A' AS Contribution504B
       ,'B' AS BasicLifePlan
-      ,sr.annual_salary
-      ,sr.position_effective_start_date
-      ,sr.termination_date
-FROM gabby.people.staff_roster sr
+
+FROM gabby.people.staff_roster s
 LEFT JOIN gabby.adp.workers_custom_field_group_wide_static cf
-  ON sr.associate_id = cf.worker_id
-WHERE sr.business_unit = 'KIPP Miami'
+  ON s.associate_id = cf.worker_id
+WHERE s.business_unit = 'KIPP Miami'
+  AND (termination_date >'2021-06-30' OR termination_date IS NULL)
```

Comment:
should this be 2022? either way use `DATEFROMPARTS()` and `GLOBAL_ACADEMIC_YEAR()`

Reply:
what i'm trying to get is only people who have null termination dates or termination dates within the academic year... is there a way to look for dates between the global academic year start and end dates?

Reply:
`DATEFROMPARTS()` can do that, you feed it `GLOBAL_ACADEMIC_YEAR` as the year